### PR TITLE
MAKE-1151: Initialize float series as empty array to avoid null JSON …

### PR DESCRIPTION
**Pull request by zamj**
…value

When marshaled, nil slices get encoded as `null` while empty arrays get encoded as `[]`.

https://golang.org/pkg/encoding/json/#Marshal

https://play.golang.org/p/jvZ7Sx9hoz2

### CODE REVIEW

File: units/recalculate_change_points.go [hunk 1]
```diff
@@ -99,7 +99,7 @@ func (j *recalculateChangePointsJob) Run(ctx context.Context) {
 		sort.Slice(series.TimeSeries, func(i, j int) bool {
 			return series.TimeSeries[i].Order < series.TimeSeries[j].Order
 		})
-		var float_series []float64
+		float_series := make([]float64, 0)
```
Comment (julianedwards) [inline units/recalculate_change_points.go]: do `float_series := []float64{}`

Comment (tychoish) [inline units/recalculate_change_points.go]: why not make([]float64, len(series.TimeSeries), and then avoid append and just set indexes....

Comment (julianedwards) [reply units/recalculate_change_points.go]: oh yeah i missed the for loop on line 103, this makes more sense

Comment (zamj) [reply units/recalculate_change_points.go]: Sure, done

**Pull request merged**